packages/core/src/external/aws/app-config.ts

```diff
@@ -20,22 +20,28 @@ export type StringValuesFF = z.infer<typeof ffStringValuesSchema>;
 
 export const ffBooleanSchema = z.object({
   enabled: z.boolean(),
-  values: z.never().or(z.literal(undefined)),
 });
 export type BooleanFF = z.infer<typeof ffBooleanSchema>;
 
-export const ffDatastoreSchema = z.object({
+export const booleanFFsSchema = z.object({
+  commonwellFeatureFlag: ffBooleanSchema,
+  carequalityFeatureFlag: ffBooleanSchema,
+});
+export type BooleanFeatureFlags = z.infer<typeof booleanFFsSchema>;
+
+export const stringValueFFsSchema = z.object({
```

Comment:
I know this is from the previous FF PR, but I was a little confused since StringValuesFF is really BooleanFF + values I had to do some digging to make sure the StringValues FF also included the enabled boolean since it was clear from the name. Could we change the stringValueFFsSchema to stringValueAndBooleanFFsSchema or is that too wordy?

Reply:
I think its not necessary. If we had 3 or 4 properties we wouldn't add all them on the name 😅